UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+              }
+            }
+          ],
+          "text": "Medicare ID",
+          "type": "string"
+        }
+      ]
+    },
+    {
+      "linkId": "PND",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "gtable"
+              }
+            ]
+          }
+        }
+      ],
+      "text": "Provider (physician/NPP) who performed the in-person evaluation if different from the signing provider",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "PND.1a",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".LastName"
+              }
+            }
+          ],
+          "text": "Last Name",
+          "type": "string"
+        },
+        {
+          "linkId": "PND.1b",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".FirstName"
+              }
+            }
+          ],
+          "text": "First Name",
+          "type": "string"
+        },
+        {
+          "linkId": "PND.1c",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".MiddleInitial"
+              }
+            }
+          ],
+          "text": "Middle Initial",
+          "type": "string"
+        },
+        {
+          "linkId": "PND.2",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".NPI"
+              }
+            }
+          ],
+          "text": "NPI",
+          "type": "string"
+        }
+      ]
+    },
+    {
+      "linkId": "ENC",
+      "text": "Encounter",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "ENC.4",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".Today"
+              }
+            }
+          ],
+          "text": "Date of encounter (MM/DD/YYYY):",
```

Comment:
Date of the evaluation is in the Provider section. If you are using the substitution, that is fine with me.